mediation-old/src/main/java/com/wso2telco/dep/mediator/impl/payment/AmountChargeHandler.java

```diff
@@ -192,12 +190,12 @@ public boolean handle(MessageContext context) throws Exception {
 		// set information to the message context, to be used in the sequence
         HandlerUtils.setHandlerProperty(context, this.getClass().getSimpleName());
         HandlerUtils.setEndpointProperty(context, sending_add);
-        HandlerUtils.setGatewayHost(context);
         HandlerUtils.setAuthorizationHeader(context, executor, endpoint);
         context.setProperty("operator", endpoint.getOperator());
         context.setProperty("requestResourceUrl", requestResourceURL);
         context.setProperty("requestID", requestId);
         context.setProperty("clientCorrelator", clientCorrelator);
+        context.setProperty("hubGateway",mediatorConfMap.get("hubGateway"));
```

Comment:
setGatewayHost method has been updated to user configFileReader. Setting hubGateway property directly to context  here makes the util method redundant.

Reply:
In the util class, reads the properties again. in here we have already required values here. 
In the performance aspect, the above suggested method has a negative affect. The better practice is using already fetched values and do the operation.

Reply:
Need to finalise implementation and usage on handlers. Leaving it like this for now as there's no functional impact.

Reply:
The above implementation is useful only when there is no loaded configuration. Otherwise no pint of using this